src/proto/FunctionRpc.proto

```diff
@@ -553,6 +553,13 @@ message RpcException {
 
   // Textual message describing the exception
   string message = 2;
+
+  // Type of exception if thrown by user's function code 
+  optional string remote_type = 4; 
```

Comment:
What does "remote_" mean in this case? And does it have to be "if thrown by user's function code"? I could see workers wanting to specify exception type for system errors, too

Reply:
These changes are specifically for errors thrown by users' code - I could come up with a more clear name.

Reply:
Maybe user? i.e. `user_stack` & `user_type` or user exception i.e. ` user_exception_type` etc.

This leaves it open for future implementation for things like a system_exception_type etc (this is out of scope for the changes Maddie is making)

Reply:
I'm still just not sure what `remote_` means in this case. Can it just be `type`? Or `type_name`?

I'm fine to keep one property, but maybe add more context to the comment. In general, I wish this protobuf file had more context in the comments, because as a language owner it's hard to know what the intent was for all these properties and how the host handles them. Something like this:

> // Type of exception. If it's a user exception the type is passed along to app insights, otherwise it's ignored for now

Reply:
That makes sense - I will add a comment for context. I think your point about just making it type_name makes sense, since there wasn't already an existing type property on the RpcException message.